src/Headphones.h

```diff
@@ -84,7 +86,18 @@ enum class HeadphonesEvent
 
 	InteractionUpdate,
 };
+
 using HeadphonesMessage = CommandSerializer::CommandMessage;
+
+enum DeviceCapabilities
```

Comment:
Maybe you meant enum class? 🤔 C enums don't need bitwise overloads

Reply:
In C they don't, but in C++ they do. This is what Microsoft does to enums in their codebase too.
And enum classes do not implicitly cast to their underlying type making me have to do something like this to test if a flag is active or not:
```cpp
(deviceCapabilities & DeviceCapabilities::DC_AutoAsm) != DeviceCapabilities::DC_None
```
Can't do this:
```cpp
(deviceCapabilities & DeviceCapabilities::DC_AutoAsm) != 0
```

(At least with the bitwise overloads I adapted from the Windows SDK)